src/app/home/components/welcome.ts

```diff
@@ -5,13 +5,15 @@ import { Component } from '@angular/core';
   template: `
     <div id="welcome" class="welcome-container centerBlock">
       <div class="welcome hidden-xs"></div>
-      <div class="welcome-title">Angular.JS Cruise</div>
+      <div class="welcome-title">Angular Cruise</div>
       <div class="welcome-sub">
-        Sub Headline Lorem Ipsum Dolor Sit<br/>
+        <p>As <a href="https://twitter.com/jeffbcross/status/776891439657525250">Jeff Cross' beard</a> has stated,</p>
+        <p>we're officially jumping the shark</p>
+        <p>May 29, 2017!</p>
         <a href="https://www.ng-conf.org/#/conduct" title="This code of conduct was adopted from ng-conf's code of conduct" target="_blank">Code of Conduct</a>
       </div>
       <div class="centerBlock">
-        <button class="main-btn grow" onclick="document.location='/#/home#tickets';return false;">Purchase</button> 
+        <button class="main-btn grow" onclick="document.location='/#/home#tickets';return false;">Purchase</button>
```

Comment:
I'm no ng2 pro--but I'm guessing this isn't the idiomatic way to route to a different page